Messaging.md

```diff
@@ -364,40 +386,42 @@ Note that `outcome_i` is the outcome value itself and not its hash that will be
 This type contains information provided by an oracle on an event that it will attest to.
 See [the Oracle specifications](./Oracle.md#oracle-event) for more details.
 
-#### Version 0 `oracle_event`
+#### `oracle_event`
 
-1. type: 55330 (`oracle_event_v0`)
-2. data:
+1. data:
    * [`u16`:`nb_nonces`]
    * [`nb_nonces*x_point`:`oracle_nonces`]
    * [`u32`:`event_maturity_epoch`]
    * [`event_descriptor`:`event_descriptor`]
    * [`string`:`event_id`]
 
-### The `oracle_announcement` Type
+### The `oracle_announcement` TLV
```

Comment:
I'm confused about the purpose of the `TLV` here

Should there be a separate `TLV` and message format? 

Currently `OracleAnnouncementV0` and `OracleAttestmentV0` are both TLVs (at least they're both implemented as TLVs in bitcoin-s and node-dlc)

Reply:
+1 if nothing else is changing about the announcements I don't see the reasoning of changing the type

Reply:
Sorry I confused myself. I think they should be `wire messages` as they can be broadcast directly (please see updated version). I guess we could define two formats but for the sake of simplicity I would personally prefer keeping a single one.

> +1 if nothing else is changing about the announcements I don't see the reasoning of changing the type

I was thinking since we are going to have breaking changes on these anyway we might as well make it so that the wire message types follow each others (I left an open space for the `close dlc`. Though now thinking about it I wonder if `close dlc` should actually have an odd type number 🤔

Reply:
> Sorry I confused myself. I think they should be wire messages as they can be broadcast directly (please see updated version). I guess we could define two formats but for the sake of simplicity I would personally prefer keeping a single one.
+1 for having one format for sake of simplicity. Keeping it as a `wire message` makes sense to me

> Though now thinking about it I wonder if close dlc should actually have an odd type number 🤔
What would be the rationale for this? I'm guessing because it's optional?

Reply:
I noticed that in the latest version of #163 `OracleAnnouncement`, `OracleEvent`, `EnumEventDescriptor` and `DigitDecompositionEventDescriptor` are all `wire messages`

This makes sense for `OracleAnnouncement`, since it's likely announcements and attestments would be communicated over the transport layer. However, shouldn't the wire messages inside `OracleAnnouncement` actually be a TLVs since it's unlikely for them to be communicated over the transport layer.

Reply:
The reason was that it was decided not to touch any oracle related message in this PR so that no update to oracle software is required for now, and then make all oracle related changes as part of #167 at once.